lib/python/rose/popen.py

```diff
@@ -225,6 +228,8 @@ def run_ok(self, *args, **kwargs):
         """
         ret_code, stdout, stderr = self.run(*args, **kwargs)
         if ret_code:
+            if isinstance(stderr, bytes):
```

Comment:
Shouldn't stdout, stderr always be bytes?

Reply:
Unless it's `None`, in which case attempting to decode it won't work.